src/fs/filter.rs

```diff
@@ -41,6 +41,9 @@ pub struct FileFilter {
     /// ones, depending on the sort field.
     pub reverse: bool,
 
+    /// Whether to only show directories
```

Comment:
Nit: this should have a period for consistency.

Reply:
Should be fixed now.